sys-fs/mtd-utils/mtd-utils-2.1.2.ebuild

```diff
@@ -0,0 +1,49 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+
+SRC_URI="https://github.com/sigma-star/mtd-utils/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+KEYWORDS="~amd64 ~arm ~mips ~ppc ~ppc64 ~x86 ~amd64-linux ~x86-linux"
+S="${WORKDIR}/${PN}-v${PV}-0ef86b1"
```

Comment:
... ```
 * ERROR: sys-fs/mtd-utils-2.1.2::testworld failed (prepare phase):
 *   The source directory '/var/tmp/portage/sys-fs/mtd-utils-2.1.2/work/mtd-utils-v2.1.2-0ef86b1' doesn't exist
```

Reply:
Something was wrong with my distfiles, that also caused the workdir error, it should be okay now.